[SOAR-2742] - Manage Expiring Vulnerability Exceptions with InsightVM (#327)
rapid7/insightconnect-workflows: workflows/Manage_Expiring_Vulnerability_Exceptions_with_InsightVM/help.md

# Description

> Create a slack message to the user weekly and notify them of any expiring exceptions.

Reviewer: ```suggestion
Notify a user weekly via Slack message of expiring exceptions.
```

# Key Features

* Quick management of exceptions with InsightVM through Slack

# Requirements

* [Slack](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops)
* InsightVM

# Documentation

## Setup

Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.

> After import, you can change the `Weekly Timer` step to adjust the date and frequency of reminders. 

Reviewer: ```suggestion
After import, adjust the date and frequency in the `Weekly Timer` step as necessary.
```

> Change the Slack to change the channel from the preset `change_me` to the channel in your environment where you would like reminders posted.

Reviewer: ```suggestion
Update the Slack steps to change the channel from the preset `change_me` to the channel in your environment where you would like reminders posted.
```
